features/organizer/get-creator.feature

```diff
@@ -2,7 +2,7 @@ Feature: Test getting creator of organizer
     Background:
         Given I am using the UDB3 base URL
         And I am using an UiTID v1 API key of consumer "uitdatabank"
-        And I am authorized as JWT provider v2 user "dev_e2e_test"
+        And I am authorized as JWT provider v2 user "udbtestinvoerder_ownerships"
```

Comment:
Is the udbtest prefix necessary?

Reply:
It's not needed, but it needs to be aligned with the appconfig.

For me it's fine to remove the `udbtest` prefix, what do you think @JonasVHG ?
If we remove it here we also need to remove it from the `udbtestinvoerder_1` and `udbtestinvoerder_2`

Reply:
I already merged, it's easy enough to create a new pull request for removing the prefix.